src/components/Food.tsx

```diff
@@ -0,0 +1,18 @@
+import { FC } from "react";
+
+import { Coordinate } from "@/models/coordinate";
+import styles from "./Food.module.scss";
+
+const Food: FC<Coordinate> = (item): React.ReactElement => {
+  return (
+    <div
+      className={styles.food}
+      style={{
+        gridRowStart: item.y,
+        gridColumnStart: item.x,
```

Comment:
Did you consider using styled components? I think it will perfectly fit the dynamic parameters you use. (just a question)